pkg/extensions/search/resolver.go

```diff
@@ -277,6 +277,135 @@ func getCVEListForImage(
 	}, nil
 }
 
+func getCVEDiffListForImages(
+	ctx context.Context, //nolint:unparam // may be used in the future to filter by permissions
+	minuend string,
+	subtrahend string,
+	metaDB mTypes.MetaDB,
+	cveInfo cveinfo.CveInfo,
+	requestedPage *gql_generated.PageInput,
+	searchedCVE string,
+	log log.Logger, //nolint:unparam // may be used by devs for debugging
+) (*gql_generated.CVEDiffResult, error) {
+	minuendRepo, minuendTag := zcommon.GetImageDirAndTag(minuend)
```

Comment:
I think we should account for the os/architecture. Maybe get the manifest digest as part of the minuend, and take the architecture of that manifest into consideration. Or have it as a parameter.

Using minuendSummary.Manifests[0] seems very shaky, and will not work for multiarch images in case a vulnerability is for a specific architecture.